src/Core/src/Animations/Animation.cs

```diff
@@ -21,51 +36,122 @@ public Animation(Action<double> callback, double start = 0.0f, double duration =
 			Step = callback;
 
 		}
+
+		/// <summary>
+		/// Instantiate a new <see cref="Animation"/> object that consists of the given list of child animations.
+		/// </summary>
+		/// <param name="animations">A <see cref="List{T}"/> that contains <see cref="Animation"/> objects that will be children of the newly instantiated animation.</param>
 		public Animation(List<Animation> animations)
 		{
 			childrenAnimations = animations;
 		}
+
 		internal WeakReference<IAnimator>? Parent { get; set; }
 
+		/// <summary>
+		/// A callback that is invoked whenever this animation finishes.
+		/// </summary>
 		public Action? Finished { get; set; }
+
+		/// <summary>
+		/// A callback that is invoked after each tick of this animation.
+		/// </summary>
 		public Action<double>? Step { get; set; }
-		bool paused;
-		public bool IsPaused => paused;
-		protected List<Animation> childrenAnimations = new List<Animation>();
 
+		bool _paused;
+
+		/// <summary>
+		/// Specifies whether this animation is currently paused.
+		/// </summary>
+		public bool IsPaused => _paused;
+
+		/// <summary>
+		/// Collection of child animations associated to this animation.
+		/// </summary>
+		protected List<Animation> childrenAnimations = new();
+
+		/// <summary>
+		/// The name of this animation.
+		/// </summary>
 		public string? Name { get; set; }
+
+		/// <summary>
+		/// The delay (in seconds) taken into account before the animation starts.
+		/// </summary>
 		public double StartDelay { get; set; }
+
+		/// <summary>
+		/// The duration of this animation.
```

Comment:
Can we specify the units for the duration? (e.g., milliseconds, seconds)

Reply:
Good call! Done!